src/js/flagset.ts

```diff
@@ -399,11 +399,9 @@ export const FLAGS: FlagSection[] = [{
     flag: 'Gt',
     name: 'Statue glitch may be required',
     text: `Progression may require glitching past guards without Change or Paralysis,
-           or people turned to stone without a Flute of Lime.  The logic <i>hopefully</i>
-           ensures that using the Flute of Lime on the two statues will not break the
-           game, but we're less confident that this is always the case, so for safety
-           we recommend always glitching past the statues if this option is set, even if
-           the first Flute of Lime has been found.`,
+           or people turned to stone without a Flute of Lime.  The logic ensures that 
```

Comment:
Maybe just leave out the second sentence and leave it at "without a Flute of Lime."

We should also add some notes in the more general documentation about the two-usages, the new alarm flute location etc.

Reply:
"We should also add some notes in the more general documentation about the two-usages, the new alarm flute location etc." - is the help section good for this? That seems to have the best list of things that "changed" from vanilla.

Reply:
Yes, though Mattrick is also taking a stab at updating it, so you might conflict.